src/Components/CoinData.jsx

```diff
@@ -1,41 +1,27 @@
 import { useState, useEffect, useContext} from 'react'
 import axios from 'axios'
+import { SearchContext } from './Contexts/searchContext';
```

Comment:
First external imports should occur(from installed libraries), then 1 line space, then internal file/component/function imports, then 1 line space and then finally css imports

Reply:
This is not resolved yet